spec/legacy_taskscheduler.rb

```diff
@@ -0,0 +1,1271 @@
+# DO NOT EDIT, ALTER, OR DELETE THIS FILE
```

Comment:
Probably want to reference which commit this file came from e.g.

https://github.com/puppetlabs/puppet/blob/5.5.3/lib/puppet/util/windows/taskscheduler.rb